contrib/comfort_models.py

```diff
@@ -171,67 +215,86 @@ def comfPMV(ta, tr, vel, rh, met, clo, wme):
+    - pmv : Predicted mean vote
+    - ppd : Percent predicted dissatisfied [%]
+    - ta_adj: Air temperature adjusted for air speed [C]
+    - cooling_effect : The difference between the air temperature and adjusted air temperature [C]
+    - set: The Standard Effective Temperature [C] (see below)
+
+    :param ta: dry bulb air temperature, [C]
+    :type  ta: float
+    :param tr: mean radiant temperature, [C]
+    :type  tr: float
+    :param vel: air velocity, [m/s]
+    :type  vel: float
+    :param rh: relative humidity, [%]
+    :type  rh: float
+    :param met: metabolic rate, [met]
+    :type  met: float
+    :param clo: clothing insulation, [clo]
+    :type  clo: float
+    :param wme: external work, [met] default 0
+    :type  wme: float
+
+    :return: list containing estimated parameters [PMV, PPD, Ta_adj, cooling_effect, SET]
+    :rtype: list
+
     """
-    res = None
 
-    def findSaturatedVaporPressureTorr(T):
-        # calculates Saturated Vapor Pressure (Torr) at Temperature T  (C)
-        return math.exp(18.6686 - 4030.183 / (T + 235.0))
+    def findSaturatedVaporPressureTorr(t):
+        """ Estimate the saturation vapor pressure in [torr]
+
+        :param t: air temperature, [C]
+        :type  t: float
 
-    # Key initial variables.
-    VaporPressure = (rh * findSaturatedVaporPressureTorr(ta)) / 100
+        :return: saturated vapour pressure, [torr]
+        :rtype: float
+        """
+        return math.exp(18.6686 - 4030.183 / (t + 235.0))
+
+    # Initial variables as defined in the ASHRAE 55-2017
+    VaporPressure = rh * findSaturatedVaporPressureTorr(ta) / 100
     AirVelocity = max(vel, 0.1)
     KCLO = 0.25
     BODYWEIGHT = 69.9
     BODYSURFACEAREA = 1.8258
     METFACTOR = 58.2
-    SBC = 0.000000056697  # Stefan-Boltzmann constant (W/m2K4)
+    SBC = 0.000000056697
     CSW = 170
     CDIL = 120
     CSTR = 0.5
 
-    TempSkinNeutral = 33.7  # setpoint (neutral) value for Tsk
-    TempCoreNeutral = 36.49  # setpoint value for Tcr
-    # setpoint for Tb (.1*TempSkinNeutral + .9*TempCoreNeutral)
+    TempSkinNeutral = 33.7
+    TempCoreNeutral = 36.8
     TempBodyNeutral = 36.49
-    SkinBloodFlowNeutral = 6.3  # neutral value for SkinBloodFlow
+    SkinBloodFlowNeutral = 6.3
 
-    # INITIAL VALUES - start of 1st experiment
     TempSkin = TempSkinNeutral
     TempCore = TempCoreNeutral
     SkinBloodFlow = SkinBloodFlowNeutral
     MSHIV = 0.0
     ALFA = 0.1
     ESK = 0.1 * met
 
-    # Start new experiment here (for graded experiments)
-    # UNIT CONVERSIONS (from input variables)
-
-    # This variable is the pressure of the atmosphere in kPa and was taken
-    # from the psychrometrics.js file of the CBE comfort tool.
     p = 101325.0 / 1000
-
     PressureInAtmospheres = p * 0.009869
     LTIME = 60
     TIMEH = LTIME / 60.0
     RCL = 0.155 * clo
 
-    FACL = 1.0 + 0.15 * clo  # INCREASE IN BODY SURFACE AREA DUE TO CLOTHING
```

Comment:
Is it intentional to remove all these comments?

Reply:
I remove the comments unintentionally since I copied the .js code to the Python module. I have restored the comments as you suggested.